libs/community/langchain_community/vectorstores/azuresearch.py

```diff
@@ -71,17 +71,6 @@
 MAX_UPLOAD_BATCH_SIZE = 1000
 
 
-class BearerTokenCredential(TokenCredential):
```

Comment:
@levalencia any reason you replaced this `BearerTokenCredential` class ? `TokenCredential` doesn't seem instanciable (see stacktrace: https://github.com/langchain-ai/langchain/issues/26216 )
 i was planning to reintroduce this wrapper class : proposed fix https://github.com/langchain-ai/langchain/pull/26995)